package.json

```
@@ -27,13 +27,14 @@
     "telegraf": "./bin/telegraf"
   },
   "scripts": {
-    "lint": "eslint .",
-    "pretest": "tsc --build docs/examples",
-    "test": "ava test/*",
-    "typedoc": "typedoc",
-    "clean": "git clean -fX docs/build/ lib/ typings/",
+    "prepare": "npm run build && npm run build:docs && mjs-entry src/index.ts > lib/index.mjs",
```

Docs aren't necessary to use the lib
```suggestion
    "prepare": "npm run build && mjs-entry src/index.ts > lib/index.mjs",
```